spatialos/UnrealGDK
# UNR-5232: Fixed RPCs crashing workers when called on incomplete entities

#### Description
Added a completeness check; this surfaced as player controllers' data is received in two steps.

#### Release note
Added a release note.

#### Tests
Ran automated tests, ran around in Tiny offloading gym - to ensure nothing breaks.

Repro I've been using for this issue:
1. Add QuitGame to PlayerController's BeginPlay
2. Use `t.MaxFps 1` console command
3. Start PIE

This sequence usually reproduced the issue in first 3 runs.
